src/test/java/seedu/address/model/person/PersonTagsContainsTagsPredicateTest.java

```diff
@@ -0,0 +1,101 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.tag.Tag;
+import seedu.address.testutil.PersonBuilder;
+
+public class PersonTagsContainsTagsPredicateTest {
```

Comment:
I am struggling to read the assert lines.
Would breaking out the PersonBuilder.build calls into a line by itself make the test cases more readable?